package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@leadhome/oxygen",
-  "version": "0.1.3",
+  "version": "0.1.4",
```

Comment:
You don't need to worry about this going forward. sem release deals with it and updates it. You can just do a pull from master once there has been a release.

Reply:
Ooooh okay cool thanks.